File: BikeShop/Domain/Model/Bike.cs
namespace Domain.Model;

public class Bike(string id, string type, DateTime rented, string customerId)
{
    public string Id { get; set; } = id;
    public string Type { get; set; } = type;
    public DateTime Rented { get; set; } = rented;
    public string CustomerId { get; set; } = customerId;
    public bool Returned { get; set; } = false;
}

File: bike-shop-front/src/model/Bike.ts
class Bike{
    Id: string;
    Type: string;
    Rented: Date;
    CustomerId: string;
    Returned: boolean;

    constructor(id: string, type: string, rented: Date, customerId: string, returned: boolean) {
        this.Id = id;
        this.Type = type;
        this.Rented = rented;
        this.CustomerId = customerId;
        this.Returned = returned;
    }
}

export default Bike;